taui/src/components/main-page.js

```diff
@@ -165,14 +180,21 @@ export default class MainPage extends React.PureComponent<Props> {
             neighborhoodBoundsExtent={p.neighborhoodBoundsExtent}
             origin={p.data.origin}
             pointsOfInterest={p.pointsOfInterest}
+            realtorListings={p.realtorListings}
             routableNeighborhoods={p.routableNeighborhoods}
+            showBHAListings={p.data.showBHAListings}
+            showRealtorListings={p.data.showRealtorListings}
             showRoutes={this._showNeighborhoodRoutes()}
             setActiveNeighborhood={p.setActiveNeighborhood}
             setPage={p.setPage}
             setEndPosition={p.updateEndPosition}
+            setShowBHAListings={p.setShowBHAListings}
             setShowDetails={p.setShowDetails}
+            setShowRealtorListings={p.setShowRealtorListings}
             setStartPosition={p.updateStartPosition}
+            showBHAListings={p.data.showBHAListings}
```

Comment:
I missed this on my pass, but running `docker-compose run --rm taui lint-js` informs me that this prop is duplicated here and on line 185, and `showRealtorListings` is also duplicated.